@here/harp-datasource-protocol/lib/TechniqueParams.ts

```diff
@@ -934,6 +940,11 @@ export interface DashedLineTechniqueParams extends BaseTechniqueParams, Polygona
      * Clip the line outside the tile if `true`.
      */
     clipping?: boolean;
+
+    /**
+     * Describes line caps type. Default is Round.
```

Comment:
Use markdown: `"Round"`.

Add doc what kind of cap types are supported.

Reply:
Done.